vAjasaneyam/mAdhyandinam/shatapatha-brAhmaNam/vaMshIdhara-pAThaH/sasvaram/01/02/4.md

```diff
@@ -3,25 +3,25 @@ title = "४"
 
 +++
 ## ०१
-इ᳘न्द्रो ह य᳘त्र व्वृत्रा᳘य व᳘ज्रं प्रजहा᳘र॥  
+इन्द्रो ह यत्र व्वृत्रा᳘य वज्रं प्रजहार ॥
```

Comment:
इयं पङ्क्तिः पुनः परीक्ष्यताम्।

Reply:
vajram now corrected,  but 3 notations missing

Reply:
I do not receive any correction unless you commit and push. 

Also, I suspect you are wrong. Please paste a screenshot from the book showing this sentence.